mapstore/localConfig.json

```diff
@@ -0,0 +1,606 @@
+{
+    "proxyUrl": {
+        "url": "proxy/?url=",
```

Comment:
Is it the security-proxy -provided service or one that is provided by mapstore ?

Reply:
pretty sure that's a mapstore internal proxy (for which there's already a `mapstore/proxy.properties` config file) - wether mapstore can use the security-proxy service is another (probably complicated) topic...

Reply:
mapstore is already behind the security-proxy. The internal proxy is there to allow cross-origin calls to servers that are not CORS enabled.
I don't know if the security-proxy can cover this use case too.

Reply:
> I don't know if the security-proxy can cover this use case too.

Yes, it does. geOrchestra applications usually use it. 
It is accessible at the same URL: `/proxy/?url=`
It would be interesting to report any issues you could encounter using it. Thanks